docs/streaming_how_to.rst

```diff
@@ -1,4 +1,4 @@
-.. _streaming_how_to:
+.. _streaming.py: https://github.com/tweepy/tweepy/blob/master/examples/streaming.py
```

Comment:
I believe the `streaming.py` file being referenced is the [source code file](https://github.com/tweepy/tweepy/blob/master/tweepy/streaming.py) and not the [example](https://github.com/tweepy/tweepy/blob/master/examples/streaming.py).
Also, for consistency, this should be a separate new link without replacing the title link.
```suggestion
.. _streaming_how_to:
.. _streaming.py: https://github.com/tweepy/tweepy/blob/master/tweepy/streaming.py
```